tests/expectations/tests/issue-1040.rs

```diff
@@ -1,7 +1,5 @@
 /* automatically generated by rust-bindgen */
 
-
 #![allow(dead_code, non_snake_case, non_camel_case_types, non_upper_case_globals)]
 
-
-pub const g_107: ::std::os::raw::c_ulonglong = 18446744073709551615;
+pub const g_107: ::std::os::raw::c_ulonglong = -1;
```

Comment:
This is because proc-macro2 casts ints as i64 if they aren't suffixed

Reply:
This will be fixed after https://github.com/rust-lang/rust/pull/49597 has landed.

Reply:
Does that mean another proc-macro2 breaking change ?

Reply:
Also this doesn't work for u32 soooo....